test/routes/api/resources.js

```diff
@@ -21,7 +21,15 @@ describe('/api/resources', () => {
       const res = response.body;
       assert.deepStrictEqual(res.length, 2);
     });
+
+    it('returns a list of Resources that belong to a given Category', async () => {
+      const response = await testSession.get('/api/resources?categoryId=2').expect(HttpStatus.OK);
```

Comment:
A category id is passed to the route as a _query parameter_ (see: the diagram of a URL- query parameters are key/value pairs that come after a ? in the url path).